src/libs/EmojiUtils.js

```diff
@@ -259,6 +259,42 @@ const getEmojiCodeWithSkinColor = (item, preferredSkinToneIndex) => {
     return code;
 };
 
+/**
+ * Extracts emojis from a given text.
+ *
+ * @param {String} text - The text to extract emojis from.
+ * @returns {Object[]} An array of emoji codes.
+ */
+function extractEmojis(text) {
+    const emojis = [];
+
+    if (!text) {
+        return emojis;
+    }
+
+    let parseEmojis = text.match(CONST.REGEX.EMOJIS);
+
+    if (!parseEmojis) {
+        return emojis;
+    }
+
+    parseEmojis = [...new Set(parseEmojis)]; // Some emojis are repeated with different skin tones
```

Comment:
```suggestion
    // Some emojis are repeated with different skin tones
    parseEmojis = [...new Set(parseEmojis)]; 
```

Please put comments on top like you will find all over the codebase.

It's good to also include some reasoning. I think we are referring to the usage of the `Set()` here? But not too sure why we want to ensure that different skin tones are ignored.